tests/integration/test_sqs.py

```diff
@@ -44,6 +44,8 @@
     }
 }
 
+NUMBER_OF_MESSAGES = 3
```

Comment:
nitpick: I don't think this necessarily needs to be a top-level constant (not critical, though)